### PR TITLE
#93 Separated the tests and preparation of the test environment.

### PR DESCRIPTION
**Pull request by AVM82**
connect to #93

### CODE REVIEW

File: src/test/java/juja/sqlcmd/DatabaseManagerTest.java [hunk 1]
```diff
@@ -10,65 +10,80 @@
 import java.sql.DriverManager;
 import java.sql.SQLException;
 import java.sql.Statement;
-import java.util.Date;
 
 import static junit.framework.TestCase.assertFalse;
```
Comment (Shtramak) [inline src/test/java/juja/sqlcmd/DatabaseManagerTest.java]: Why you're using this package for assertFalse? It should be org.junit.Assert.assertFalse

**Review by Shtramak — APPROVED** [review]
Good job!

File: src/test/java/juja/sqlcmd/DatabaseManagerTest.java (hunk 1 above)
Comment (AVM82) [reply src/test/java/juja/sqlcmd/DatabaseManagerTest.java]: Good remark, I'll fix it.

File: src/test/java/juja/sqlcmd/DatabaseManagerTest.java [hunk 2]
```diff
@@ -10,65 +10,80 @@
...
 
     private DatabaseManager databaseManager;
 
     @BeforeClass
     public static void createConnection() throws SQLException, ClassNotFoundException {
         Class.forName("org.postgresql.Driver");
-        connection = DriverManager.getConnection(TEST_DB_CONNECTION_URL, USER, PASSWORD);
-        dbName = new Date().toString();
+        connection = DriverManager.getConnection(TEST_DB_CONNECTION_URL, LOGIN_FOR_SU, PASSWORD_FOR_SU);
         try (Statement statement = connection.createStatement()) {
```
Comment (kortov) [inline src/test/java/juja/sqlcmd/DatabaseManagerTest.java:36]: Why don't you use your own "executeQuery" method?

File: src/test/java/juja/sqlcmd/DatabaseManagerTest.java [hunk 3]
```diff
@@ -162,30 +177,30 @@ public void testInsertWhenDataSetHasExtraColumns() throws SQLException {
...
         assertFalse(actual);
     }
 
     @Test
     public void testDeleteWhenValidData() throws SQLException {
         createTableWithData(TABLE_NAME);
-        databaseManager.connect(dbName, USER, PASSWORD);
+        databaseManager.connect(NAME_DB_TEST, LOGIN_FOR_TEST, PASSWORD_FOR_TEST);
         boolean actual = databaseManager.delete(TABLE_NAME, 1);
         executeQuery(String.format("DROP TABLE IF EXISTS %s", TABLE_NAME));
```
Comment (kortov) [inline src/test/java/juja/sqlcmd/DatabaseManagerTest.java]: You drop table twice: in the method and in the @after

File: src/test/java/juja/sqlcmd/DatabaseManagerTest.java [hunk 4]
```diff
@@ -162,30 +177,30 @@ public void testInsertWhenDataSetHasExtraColumns() throws SQLException {
...
         assertFalse(databaseManager.delete("noTable", 1));
     }
 
     @Test
     public void testDeleteWhenIdNotExists() throws SQLException {
         createTableWithData(TABLE_NAME);
-        databaseManager.connect(dbName, USER, PASSWORD);
+        databaseManager.connect(NAME_DB_TEST, LOGIN_FOR_TEST, PASSWORD_FOR_TEST);
         boolean actual = databaseManager.delete(TABLE_NAME, -1);
         executeQuery(String.format("DROP TABLE IF EXISTS %s", TABLE_NAME));
```
Comment (kortov) [inline src/test/java/juja/sqlcmd/DatabaseManagerTest.java]: You drop table twice: in the method and in the @after

File: src/test/java/juja/sqlcmd/DatabaseManagerTest.java (hunk 2 above)
Comment (Shtramak) [reply src/test/java/juja/sqlcmd/DatabaseManagerTest.java:36]: Well it's not possible while method executeQuery() is not static... And why it's not static - that is next question. As for me this util method should be static, why to create it per each instance?

File: src/test/java/juja/sqlcmd/DatabaseManagerTest.java (hunk 3 above)
Comment (Shtramak) [reply src/test/java/juja/sqlcmd/DatabaseManagerTest.java]: good remark. I agree

Comment (AVM82) [reply src/test/java/juja/sqlcmd/DatabaseManagerTest.java]: So it is. :)
I will fix it.

File: src/test/java/juja/sqlcmd/DatabaseManagerTest.java (hunk 2 above)
Comment (AVM82) [reply src/test/java/juja/sqlcmd/DatabaseManagerTest.java:36]: Thanks for the comments, I'll think about it.
Perhaps in the next PR I will do service methods, such as executeQuery(), static

**Review by adaleorion — APPROVED** [review]

**Pull request merged**